index.md

```diff
@@ -6,17 +6,17 @@ weight: 0
 ![Web interface screenshot]({{ site.github.url }}/public/img/web-screenshot.png)
 
 Zipkin is a distributed tracing system. It helps gather timing data needed to
-troubleshoot latency problems in microservice architectures. It manages both the
+troubleshoot latency problems in micro-service architectures. It manages both the
 collection and lookup of this data through a Collector and a Query service.
 Zipkin’s design is based on the
 [Google Dapper](http://research.google.com/pubs/pub36356.html) paper.
 
-Collecting traces helps developers gain deeper knowledge about how certain
-requests perform in a distributed system. Let’s say we’re having problems with
-user requests timing out. We can look up traced requests that timed out and
-display it in the web UI. We’ll be able to quickly find the service responsible
+Collecting traces helps developers see how
```

Comment:
I think this section should be completely re-written. I think the goal is to elaborate the use cases of zipkin:

Applications are instrumented to report timing data (spans) to zipkin. The Zipkin UI presents in a way that helps establish root cause when troubleshooting latency problems. For example, users can pinpoint what percentage of time a particular service caused in recent requests, filtering with relevant criteria such as http.method. The Zipkin UI also presents a Dependency diagram, showing how many traced requests went through each service.


That said, once a production system contends with real concurrency or splits into many services, crucial (and formerly easy) tasks become difficult: user-facing latency optimization, root-cause analysis of backend errors, communication about distinct pieces of a now-distributed system, etc.